src/Service/SettingsManager.php

```diff
@@ -64,6 +65,7 @@ public function __construct(
                 $this->find($results, 'KBIN_MERCURE_ENABLED', FILTER_VALIDATE_BOOLEAN) ?? true,
                 $this->find($results, 'KBIN_FEDERATION_PAGE_ENABLED', FILTER_VALIDATE_BOOLEAN) ?? $this->kbinFederationPageEnabled,
                 $this->find($results, 'KBIN_ADMIN_ONLY_OAUTH_CLIENTS', FILTER_VALIDATE_BOOLEAN) ?? $this->kbinAdminOnlyOauthClients,
+                $this->find($results, 'KBIN_PRIVATE_INSTANCE', FILTER_VALIDATE_BOOLEAN) ?? $this->kbinPrivateInstance,
```

Comment:
We've stopped setting env vars in place of defaulting here, this is because new env vars added to this (oauth is fine because they don't actually get fetched unless enabled) break existing server owners

Can see https://github.com/MbinOrg/mbin/pull/448 for reference of removing it and defaulting. So here I would expect the right side to be `?? false` and then the `.example`/docker exmaple, `config/services.yaml` to be removed. The rest should be fine, as it all calls into here and will just default false

Also, a nitpick, but we try to add new settings as `MBIN` rather than `KBIN`. We can't rename the old ones as it would break everyone running an instance

Reply:
I renamed the variable and removed the need to set it, tested it and it doesn't break the backend not setting the env variable